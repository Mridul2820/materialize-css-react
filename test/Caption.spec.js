import React from 'react';
import { render } from '@testing-library/react';
import Caption from '../src/Caption';

describe('<Caption />', () => {
  test('renders', () => {
    const { container } = render(
      <Caption placement="left">
        <h3>This is our big Tagline!</h3>
        <h5 className="light grey-text text-lighten-3">
          Here's our small slogan.
        </h5>
      </Caption>
    );

    expect(container).toMatchSnapshot();
  });
});
