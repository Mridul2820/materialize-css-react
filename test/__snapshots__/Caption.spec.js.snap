// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Caption /> renders 1`] = `
<div>
  <div
    class="caption left-align"
  >
    <h3>
      This is our big Tagline!
    </h3>
    <h5
      class="light grey-text text-lighten-3"
    >
      Here's our small slogan.
    </h5>
  </div>
</div>
`;
